io.github.carterbox.Tofu.yml

```
@@ -0,0 +1,57 @@
+app-id: io.github.carterbox.Tofu
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  # Flutter needs clang to compile Linux desktop apps
+  - org.freedesktop.Sdk.Extension.llvm15
+command: tofu
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
```

```suggestion
  - --device=dri
  - --share=ipc
```

this is needed either way.